deployment/cloudways.rst

```diff
@@ -0,0 +1,158 @@
+.. index::
+   single: Deployment; Deploying to Cloudways Cloud Servers
+
+Deploying to Cloudways 
+======================
+
+Cloudways is a managed hosting provider for PHP based applications. This step by
+step article describes how to install Symfony on Cloudways servers and how to
+deploy Symfony applications using Git and SSH.
+
+Step 1: Prepare your server and application
+-------------------------------------------
+
+`Sign up`_ or login to your **Cloudways account** and create a new server by
+selecting PHP Stack from the **Select your Application** drop-down. Also, name
+your application, server and project accordingly.
+
+.. image:: /_images/deployment/cloudways/select-application.png
+   :alt: Select PHP Stack in Drop down
+
+Step 2: Select Cloud Provider
+-----------------------------
+
+Now select your favorite cloud provider. You can select any one from the five
```

Comment:
should we just write "Select your favorite cloud provider from the list of available providers."? This way we won't have to update the article every time a new provider gets added.